docs/algorithms/cost_generic_resub.rst

```diff
@@ -0,0 +1,25 @@
+.. _cost_generic_resub:
```

Comment:
When adding a new rst file, you also need to add a line in the corresponding index file, otherwise this page is not linked to from anywhere. In your case, it fits into `index_restructuring.rst`.